.cspell.json

```diff
@@ -61,5 +61,6 @@
       "name": "embedded-svg-source",
       "pattern": "data:image\\/svg\\+xml;.+\\w+"
     }
-  ]
```

Comment:
Custom dictionary entries should be added to
https://github.com/Esri/calcite-design-system/blob/ded0b8f3b765986d7ee8793386ebbba1052807c4/support/dictionaries/dict-calcite-design-system.txt.

Please add new entries near the bottom under the `NPM package` section, keeping them in alphabetical order and following the existing structure.